app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,18 @@
+class ReviewsController < ApplicationController
+
+  before_action :require_user
+  
+  def create
+    @video = Video.find(params[:id])
+    @review = @video.reviews.build(params.require(:review).permit(:rating, :body))
+    @review.user = current_user
```

Comment:
Do you mean place it under a private method eg: review_params? Or are you suggesting merge line 6 and 7, or 7 and 8?